tmt/base.py

```diff
@@ -3048,6 +3050,17 @@ def __init__(self,
         self._environment_from_options: Optional[EnvironmentType] = None
         self.remove = self.opt('remove')
 
+    @tmt.utils.cached_property
+    def host(self) -> 'tmt.steps.provision.local.GuestLocal':
+        import tmt.steps.provision.local
+
+        return tmt.steps.provision.local.GuestLocal(
+            data=tmt.steps.provision.GuestData(
+                guest='localhost',
+                role=None),
+            name='tmt host',
```

Comment:
So... guest and host are sometimes used for both system under test and the control box. In order to prevent confusion and make it a bit more clear would `runner` be a better choice?

Reply:
Done in 199351da